cmd/rpc.go

```diff
@@ -37,6 +39,13 @@ func RPCFlags() *flag.FlagSet {
 		"Authorization token",
 	)
 
+	fset.DurationVar(
+		&timeoutFlag,
+		"timeout",
+		0,
+		"Timeout for RPC requests (e.g. 30s, 1m)",
```

Comment:
Could you describe how the user should interact with the flag? In what units the timeout should be passed, bc duration represents time in nanoseconds.?